### PR TITLE
[DX-1649] Update MDCB Control Plane Improvement

### PR DESCRIPTION
**Pull request by dcs3spp**
### For internal users - Please add a Jira DX PR ticket to the subject!

[DX-1649](https://tyktech.atlassian.net/browse/DX-1649)

---

### Preview Link

[preview](https://deploy-preview-5321--tyk-docs.netlify.app/docs/nightly/tyk-multi-data-centre/setup-controller-data-centre#gateway-configuration)

### Description

PR raised to refactor [MDCB Control Plane Gateway Configuration](https://tyk.io/docs/tyk-multi-data-centre/setup-controller-data-centre/#gateway-configuration) step 5.

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

___

### **PR Type**
documentation

___

### **Description**
- Enhanced the documentation for MDCB Control Plane Gateway Configuration by updating the JSON example to include new fields under `event_options`.
- Provided detailed explanations for the `hybrid_enabled` and `event_options` fields, clarifying their purpose and usage.
- Included examples to illustrate the functionality of the `event_options` configuration, such as event notification mechanisms and propagation settings.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>setup-controller-data-centre.md</strong><dd><code>Enhance MDCB Control Plane Gateway Configuration Documentation</code></dd></summary>
<hr>

tyk-docs/content/tyk-multi-data-centre/setup-controller-data-centre.md

<li>Updated JSON example to include new <code>event_options</code> fields.<br> <li> Added detailed explanation for <code>hybrid_enabled</code> and <code>event_options</code> <br>fields.<br> <li> Clarified the functionality of <code>event_options</code> with examples.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5321/files#diff-a168cad82858a12f3ee02b0fdb81c0caa73316b3fdd266dd97a5bf016e99973f">+13/-8</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

[DX-1649]: https://tyktech.atlassian.net/browse/DX-1649?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-docs/content/tyk-multi-data-centre/setup-controller-data-centre.md [hunk 1]
```diff
@@ -409,11 +409,10 @@ New fields are between the `...` .
...
+In the example above it can be seen that the `hybrid_enabled` and `event_options` configuration fields have been added:
 
-`event_options:` Enables key events such as updates and deletes, to be propagated to the various instance zones. API Definitions and Policies will be propagated by default, as well as the Redis key events, meaning that hashed and not hashed key events will be propagated by default in Redis and any config related to `hashed_key_event.redis` or `key_event.redis` will not be taken into consideration.
+- `hybrid_enabled:` Allows a worker gateway to login as an organization member into MDCB.
+- `event_options:` Allows Redis key events to be configured and propagated to various instance zones.The `hashed_key_event` configuration object enables propagation of updates and deletes of hashed Redis keys to various instance zones. Conversely, the `key_event` object enables propagation of updates and deletes of non hashed Redis keys. Please note that `key_event` and `hashed_key_event` take precedence over their counterparts, `key_event.redis` and `hashed_key_event.redis`. Furthermore, API Definitions and Policies are propagated irrespectively.
```
Comment (sredxny) [inline tyk-docs/content/tyk-multi-data-centre/setup-controller-data-centre.md]: redis events are not longer required to be configured int his way, right now the events related to hashed keys and non hashed keys are propagated by default. In other hand, this is required to be set for oauth tokens: revoke single oauth token and revoke all tokens.

Comment (dcs3spp) [reply tyk-docs/content/tyk-multi-data-centre/setup-controller-data-centre.md]: Thanks @sredxny can you update PR with suggestive content to reflect correct configuration for current release?

Comment (dcs3spp) [reply tyk-docs/content/tyk-multi-data-centre/setup-controller-data-centre.md]: @caroltyk @sredxny I have updated PR as discussed, could this be reviewed and amended if necessary to reflect correct configuration for current release? Is there existing config documentation that can be linked to from this section for fields in event_options?

File: tyk-docs/content/tyk-multi-data-centre/setup-controller-data-centre.md [hunk 2]
```diff
@@ -385,19 +385,18 @@ curl $DASH_URL/admin/organisations/$ORG_ID -H "Admin-Auth: $DASH_ADMIN_SECRET" |
 5. Open `myorg.json` in your favorite text editor and add the following fields as follows.
 New fields are between the `...` .
 
-```json
+```json {linenos=table,hl_lines=["6-13"],linenostart=1}
```
Comment (carolwkc) [inline tyk-docs/content/tyk-multi-data-centre/setup-controller-data-centre.md]:
```suggestion
```json {linenos=table,hl_lines=["5-12"],linenostart=1}
```

**Review by sredxny — APPROVED** [review]

**Comment by dcs3spp** [comment]
Thanks for peer approval @sredxny , @caroltyk are requested changes ok with you?

**Review by carolwkc — APPROVED** [review]
Thank you @dcs3spp /. LGTM

**Pull request merged**